internal/scorecard/alpha/tests/test_bundle.go

```diff
@@ -43,50 +39,15 @@ func GetBundle(bundlePath string) (cfg TestBundle, err error) {
 
 	// TODO evaluate another API call that would support the new
 	// bundle format
-	_, cfg.Bundles, cfg.BundleErrors = manifests.GetManifestsDir(bundlePath)
-
-	// get CRs from CSV's alm-examples annotation, assume single bundle
-	cfg.CRs = make([]unstructured.Unstructured, 0)
+	var bundles []*registry.Bundle
+	//var bundleErrors []errors.ManifestResult
+	_, bundles, _ = manifests.GetManifestsDir(bundlePath)
 
-	if len(cfg.Bundles) == 0 {
-		return cfg, fmt.Errorf("no bundle found")
-	}
-
-	csv, err := cfg.Bundles[0].ClusterServiceVersion()
+	bundle = *bundles[0]
```

Comment:
Need to check `len(bundles)` before taking index and do a `nil` check before dereferencing.

Reply:
good catch, updated to make those checks.